scripts/package.sh

```diff
@@ -82,6 +82,26 @@ if [[ "${HOST_TRIPLE}" != "x86_64-pc-windows-msvc" ]] ; then
     #cp -R rust/build/${HOST_TRIPLE}/llvm/lib/python* deploy/llvm/lib/
 fi
 
+# Sign macOS binaries
+if [[ $HOST_TRIPLE == *apple-darwin* ]]; then
+    LLVM_BIN="./deploy/llvm/bin"
+    RUST_BIN="./deploy/rust/bin"
+    RUST_LIB="./deploy/rust/lib"
+    RUST_LIB_BIN="$RUST_LIB/rustlib/aarch64-apple-darwin/bin"
+
+    ../scripts/sign.sh \
+        "$LLVM_BIN/llvm-objdump" \
+        "$LLVM_BIN/llvm-ar" \
+        "$LLVM_BIN/llvm-readobj" \
+        "$LLVM_BIN/llvm-objcopy" \
+        "$RUST_BIN/rustdoc" \
+        "$RUST_BIN/cargo" \
+        "$RUST_LIB/librustc_driver-b4e91886a4c059a0.dylib" \
+        "$RUST_LIB/libstd-6eff127b55c063c2.dylib" \
+        "$RUST_LIB_BIN/rust-lld"
+       # "$RUST_BIN/rustc" # Not signing 'rustc' duo to failing cargo build
```

Comment:
perhaps it is safer to not sign anything until we understand what is the cause of the problem. 
Perhaps signing other binaries creates a problem we just don't see yet.

Reply:
it might be the case, we are signing the gambit binary for some time now without any issues, on the other hand i'm not sure its the right comparison.
we can comment it out for now until we feel more comfortable with it of course.

Reply:
I had people use the unsigned binaries from CI and I know that they work well. I am worried that if signed binaries have some problem that only appears under some conditions, it will be hard to debug since the developers do not use the same binaries and thus will not be able to reproduce the problem. 

Since until we are able to sign all binaries, we still will give instructions on how to manually quarantine using xattr, it is safer to not sign.

I will try to setup local signing key and see if I can reproduce the problem using local build and also look if there are instructions on how to sign the rustc.

Reply:
specifically, we ask users to do this
```
sudo xattr -rd com.apple.quarantine $HOME/platform-tools-certora
```